blocks/library/table2/index.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { registerBlockType, query as hpq } from '../../api';
+import TableBlock from './table-block';
+import BlockControls from '../../block-controls';
+import BlockAlignmentToolbar from '../../block-alignment-toolbar';
+
+const { children } = hpq;
+
+registerBlockType( 'core/table2', {
+	title: wp.i18n.__( 'TinyMCE Table' ),
+	icon: 'editor-table',
+	category: 'formatting',
+
+	attributes: {
+		content: children( 'table' ),
+	},
+
+	defaultAttributes: {
+		content: [
+			<tbody key="1">
+				<tr><td><br /></td><td><br /></td></tr>
+				<tr><td><br /></td><td><br /></td></tr>
+			</tbody>,
+		],
+	},
+
+	getEditWrapperProps( attributes ) {
+		const { align } = attributes;
+		if ( 'left' === align || 'right' === align || 'wide' === align ) {
+			return { 'data-align': align };
+		}
+	},
+
+	edit( { attributes, setAttributes, focus, setFocus } ) {
```

Comment:
There's a new className prop we should add to the root element of the `edit` function (see other blocks).